src/bin/nydus-image/main.rs

```diff
@@ -532,6 +532,16 @@ fn prepare_cmd_args(bti_string: String) -> ArgMatches<'static> {
                 .required(false)
                 .global(true),
         )
+        .arg(
+            Arg::with_name("log-rotate-size")
+                .long("log-rotate-size")
```

Comment:
There is no need to bring log rotation to `nyuds-image` since it is always one-time execution.

Reply:
Ok, i have removed it.